.changelog/22614.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
+resource/aws_s3_bucket: The `website`, `website_domain`, and `website_endpoints` arguments have been deprecated and are now read-only. Use the `aws_s3_bucket_website_configuration` resource instead.
+```
```

Comment:
```suggestion
```release-note:breaking-change
resource/aws_s3_bucket: The `website`, `website_domain`, and `website_endpoint` arguments have been deprecated and are now read-only. Use the `aws_s3_bucket_website_configuration` resource instead.
```
```